tests/integration_tests/datasets/api_tests.py

```diff
@@ -1842,3 +1842,30 @@ def test_get_datasets_is_certified_filter(self):
 
         db.session.delete(table_w_certification)
         db.session.commit()
+
+    @pytest.mark.usefixtures("create_datasets")
+    def test_get_dataset_samples(self):
+        """
+        Dataset API: Test get dataset samples
+        """
+        dataset = self.get_fixture_datasets()[0]
+
+        self.login(username="admin")
+        uri = f"api/v1/dataset/{dataset.id}/samples"
+        # feeds data
+        self.client.get(uri)
+        # get from cache
+        rv = self.client.get(uri)
+        rv_data = json.loads(rv.data)
+        assert rv.status_code == 200
+        assert "samples" in rv_data
+        assert rv_data["samples"]["cached_dttm"] is not None
```

Comment:
Thought: I think it would be a good idea to also assert that the response contains actual rows from the table. But looking at the chart data endpoint integration tests they don't seem to test for actual rows, so maybe we should consider making those tests more comprehensive in a follow-up PR (or alternatively adding more comprehensive tests here if we do decide do deprecate sampling support from the chart data endpoint).

Reply:
It seems that have not a standardized dataset for the data integrality test now. I will add a assert to test data field.

Reply:
I think a good enough test for now could be just asserting that the first row in a dataset that's contained in the `create_datasets` fixture contains the correct columns.

Reply:
done, I have a test case for this.